dist/tools/pr_check/pr_check.sh

```diff
@@ -25,7 +25,7 @@ else
 fi
 
 SQUASH_COMMITS="$(git log $(git merge-base HEAD "${RIOT_MASTER}")...HEAD --pretty=format:"    %h %s" | \
-                  grep -i -e "^    [0-9a-f]\{7\} .\{0,2\}SQUASH" -e "^    [0-9a-f]\{7\} .\{0,2\}FIX")"
+                  grep -i -e "^    [0-9a-f]\{7,9\} .\{0,2\}SQUASH" -e "^    [0-9a-f]\{7,9\} .\{0,2\}FIX")"
```

Comment:
Better to make it flexible let it range either `+` (1 to many) or 7 to 40 (full SHA-1 hash)